app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -48,80 +48,9 @@ def postpone_email_change_until_confirmation_and_regenerate_confirmation_token
     generate_confirmation_token
   end
 
-  def email
-    get_encrypted_attribute(name: :email, default: '')
-  end
-
+  # Override usual setter method in order to also set fingerprint
   def email=(email)
-    set_encrypted_attribute(name: :email, value: email, default: '')
+    set_encrypted_attribute(name: :email, value: email)
```

Comment:
So the `default` param no longer required? I will confess I am shocked by that, as I had tests failing all over when I did not distinguish between `nil` and `''` for the default `email` value.